### PR TITLE
Jewel daemon

### LINKED ISSUE #254: Add daemon for jewel
I have an experimental working version for `jewel` at https://github.com/ivotron/ceph-docker/tree/jewel . I've only tested this as a mon, osd and mds. This has only one commit https://github.com/ivotron/ceph-docker/commit/b072724935e23ad90ad56155cd250a6dc9de0d9c

### PR DESCRIPTION
**Pull request by ivotron**
fixes #254

### CODE REVIEW

File: base/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+# CEPH VERSION: Jewel
+# CEPH VERSION DETAIL: 10.2.x
 
-FROM ubuntu:14.04
+FROM ubuntu:trusty
 MAINTAINER Sébastien Han "seb@redhat.com"
 
-ENV ETCDCTL_VERSION v2.2.4
+ARG DEBIAN_FRONTEND=noninteractive
+ENV ETCDCTL_VERSION v2.0.10
```
Comment (Ulexus) [inline base/Dockerfile]: Is there a reason to regress the etcdctl version here?

File: base/Dockerfile [hunk 2]
```diff
@@ -18,12 +18,12 @@ ADD https://github.com/AcalephStorage/kviator/releases/download/v${KVIATOR_VERSI
 # Download confd
 ADD https://github.com/kelseyhightower/confd/releases/download/v${CONFD_VERSION}/confd-${CONFD_VERSION}-linux-amd64 /usr/local/bin/confd
 
-# Install prerequisites
-RUN apt-get update &&  apt-get install -y wget unzip uuid-runtime && \
+# install prerequisites
+RUN apt-get update &&  apt-get install -y wget unzip uuid-runtime python-setuptools udev && \
```
Comment (Ulexus) [inline base/Dockerfile]: I take it udev is a requirement somewhere?  The idea of installing udev in a container is... not pleasant.

Comment (ivotron) [inline base/Dockerfile]: `ceph-disk` invokes `udev` and `udevadm` (e.g. [here](https://github.com/ceph/ceph/blob/master/src/ceph-disk/ceph_disk/main.py#L1399))

File: base/Dockerfile (hunk 1 above)
Comment (ivotron) [inline base/Dockerfile]: sorry, I believe I was basing this on an outdated version. I force-pushed a commit without this change.

**Comment by Ulexus** [comment]
lgtm

**Comment by ivotron** [comment]
awesome, thanks a lot!

File: base/Dockerfile (hunk 2 above)
Comment (ktdreyer) [inline base/Dockerfile]: @ivotron , is the ceph-osd .deb package already pulling this in as a dependency?

If something in Ceph depends on something else, I want it to be fixed in the packaging.

File: base/Dockerfile [hunk 3]
```diff
@@ -18,12 +18,12 @@ ADD https://github.com/AcalephStorage/kviator/releases/download/v${KVIATOR_VERSI
...
 \
-# Install Ceph
+# install ceph
     wget -q -O- 'https://download.ceph.com/keys/release.asc' | apt-key add - && \
-    echo deb http://ceph.com/debian-${CEPH_VERSION}/ trusty main | tee /etc/apt/sources.list.d/ceph-${CEPH_VERSION}.list && \
+    echo "deb http://download.ceph.com/debian-$CEPH_VERSION/ trusty main" | tee /etc/apt/sources.list.d/ceph-$CEPH_VERSION.list && \
     apt-get update && apt-get install -y --force-yes ceph radosgw && \
```
Comment (leseb) [inline base/Dockerfile]: @ktdreyer shouldn't this be `ceph-mon`, `ceph-osd`, `ceph-mds` with Jewel?

Comment (ktdreyer) [inline base/Dockerfile]: Yes, that is correct for Jewel.

Comment (ivotron) [inline base/Dockerfile]: Should we fix this on this branch or in master/ceph-releases/jewel ?

Comment (leseb) [inline base/Dockerfile]: Everything should now go into  `master/ceph-releases/jewel` :)

**Pull request merged**